lib/rubocop/cop/metrics/collection_literal_length.rb

```diff
@@ -52,12 +52,19 @@ class CollectionLiteralLength < Base
               'Prefer reading the data from an external source.'
         RESTRICT_ON_SEND = [:[]].freeze
 
+        # @!method set?(node)
+        def_node_matcher :set?, <<~PATTERN
```

Comment:
That' my two cents. Can you rename to the following?

```suggestion
        # @!method set_const?(node)
        def_node_matcher :set_const?, <<~PATTERN
```

Reply:
Sure thing, updated.